### PR TITLE
PAE-791 - Add some tweaks to the Pathways LMS theme.

### PR DESCRIPTION
**Pull request by Squirrel18**
## Description:

This PR adds multiple tweaks to the Pathways LMS theme according to the mockups. 

## Screenshots:

**Before:**
image

**After:**
image

**Before:**
image

**After:**
image

**Before:**
image

**After:**
image

## Reviewers:

- [ ] @diegomillan 
- [ ] @ivanvgh

### CODE REVIEW

File: edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_variables.scss [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
 //color
-$primary: #007FA3;
+$primary: #005A70;
 $inverse: #FFF;
 $secondary: #ffbd3c;
 $gray: #dbdbdb;
 $footer-background: #0d2b52;
 $footer-contrast: rgba(255,255,255,0.7);
 $dark: #313131;
+$gray: #E6E6E6;
```
Comment (diegomillan) [inline edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_variables.scss:10]: I think this is an override for `$gray` at line 7. Could  we use just one variable? @Squirrel18

**Review by diegomillan — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-pathways-theme/lms/templates/dashboard/_dashboard_course_listing.html [hunk 2]
```diff
@@ -153,9 +153,19 @@ <h3 class="course-title" id="course-title-${enrollment.course_id}">
...
-                        <span class="info-date-block">${container_string.format(date=course_date)}</span>
+                      <span class="info-date-block">${container_string.format(date=course_date)}</span>
                     % elif course_date is not None:
-                        <span class="info-date-block localized-datetime" data-language="${user_language}" data-timezone="${user_timezone}" data-datetime="${course_date.strftime('%Y-%m-%dT%H:%M:%S%z')}" data-format=${format} data-string="${container_string}"></span>
+                      <%
+                        # Formatting the date string like this: Started - <b>{date}</b>
+                        container_string = Text(container_string).format(
+                          date=Text('{bold_start}{date}{bold_end}').format(
+                            bold_start=HTML('<b>'),
+                            date=course_date.strftime('%b %-d, %Y'),
```
Comment (ivanvgh) [inline edx-platform/pearson-pathways-theme/lms/templates/dashboard/_dashboard_course_listing.html:163]: @Squirrel18, just to know, what is the difference?
```suggestion
                            date=course_date.strftime('%b %d, %Y'),
```

File: edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_variables.scss (hunk 1 above)
Comment (Squirrel18) [reply edx-platform/pearson-pathways-theme/lms/static/sass/partials/lms/theme/_variables.scss:10]: Addressed. @diegomillan

File: edx-platform/pearson-pathways-theme/lms/templates/dashboard/_dashboard_course_listing.html (hunk 2 above)
Comment (Squirrel18) [reply edx-platform/pearson-pathways-theme/lms/templates/dashboard/_dashboard_course_listing.html:163]: @ivanvgh Removes the zero-padded value. https://stackoverflow.com/questions/904928/python-strftime-date-without-leading-0

**Review by diegomillan — APPROVED** [review]

File: edx-platform/pearson-pathways-theme/lms/templates/dashboard/_dashboard_course_listing.html (hunk 2 above)
Comment (ivanvgh) [reply edx-platform/pearson-pathways-theme/lms/templates/dashboard/_dashboard_course_listing.html:163]: Thanks.

**Review by ivanvgh — APPROVED** [review]

**Pull request merged**